features/support/ui/your_contributions.rb

```diff
@@ -8,6 +8,11 @@ class YourContributions < SitePrism::Page
     set_url '{/locale}/tools/wpcc/your_contributions'
     set_url_matcher(/wpcc\/\d+\/your_contributions/)
 
+    section :employee_percent,   EmployeePercentSection, '.contributions__source--employee'
+    section :employer_percent,   EmployerPercentSection, '.contributions__source--employer'
+    
     element :details, '.details__row'
+    element :contributions_description, '.t-contributions_description'
```

Comment:
There was no easy way to target the specific paragraph tag. It did not have a class and was not unique within its parent tag. I used the mas de facto standard which prefaces a class name with `t-` to indicate its used for test purposes only. 

I'm not a fan of this standard and have had a brief convo with @Tr4pSt3R about it. I thought about targetting it by looking for the first `.contribution-row` within `.contributions`. Neither option is ideal and imo too tightly couples the tests with the css.  

So I opted for the easier option.  Happy to hear others views on this - @tomas-stefano @Tr4pSt3R